### DISCUSSION TITLE
custom  blogTagsPostsComponent get an error

### DISCUSSION THREAD

**Discussion by pincman** [facebook/docusaurus#7915] [Q&A]
` pnpm swizzle @docusaurus/theme-classic BlogTagsPostsPage --typescript --eject --danger ` and `pnpm build` get an error `server-side rendering could not render static page with path xxx`

**Comment by slorber** [comment]
We can't help with so little information. Please submit a repro. Try with npm/yarn and see if it works better. Don't change anything on the swizzled component to make sure it's not your changes causing the problem.

This works on our own website so I suspect you caused the problem with your custom code:


```js
yarn workspace website swizzle @docusaurus/theme-classic BlogTagsPostsPage --typescript --eject --danger
yarn build:website:fast
```
